src/robot/client.ts

```diff
@@ -407,6 +414,10 @@ export class RobotClient extends EventDispatcher implements Robot {
         },
       };
 
+      if (priority && opts.webrtcOptions) {
```

Comment:
isn't opts.webrtcOptions going to always evaluate to true? if so we should just remove it as part of the if statement

Reply:
Yes, when you read the code, but for some reason Typescript says that it could be undefined